data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/BeaconRestApiTypes.java

```diff
@@ -62,6 +64,18 @@ public class BeaconRestApiTypes {
           .format("string")
           .build();
 
+  public static final DeserializableTypeDefinition<BLSPublicKey> PUBKEY_TYPE =
+      DeserializableTypeDefinition.string(BLSPublicKey.class)
+          .name("PubKey")
+          .formatter(BLSPublicKey::toString)
+          .parser(value -> BLSPublicKey.fromBytesCompressedValidate(Bytes48.fromHexString(value)))
+          .pattern("^0x[a-fA-F0-9]{96}$")
+          .example(
+              "0x93247f2209abcacf57b75a51dafae777f9dd38bc7053d1af526f220a7489a6d3a2753e5f3e8b1cfe39b56f43611df74a")
+          .description(
+              "The validator's BLS public key, uniquely identifying them. _48-bytes, hex encoded with 0x prefix, case insensitive._")
+          .build();
```

Comment:
might be worth seeing if this can be defined in CoreTypes, and used where ValidatorTypes is using it as well, appears to be the same definition...

Reply:
I can't use `BLSPublicKey` in `CoreTypes`. That would require a dependency to be added. Should I try that or just leave it how it is?

Reply:
leave it for now and maybe log a ticket to clean it up...